### PR TITLE
[bitnami/metrics-server] Support configuring service labels

### LINKED ISSUE #3840: [bitnami/metrics-server] Support configuring service labels
**Which chart**:
bitnami/metrics-server = 4.3.2

**Is your feature request related to a problem? Please describe.**
The stable chart allows changing the the labels of a service with `service.labels` but the Bitnami does not, making the transition harder.

**Describe the solution you'd like**
To allow `service.labels` to be configured via `values.yaml`.

### PR DESCRIPTION
**Pull request by jiwuh**
**Description of the change**

Added support for configuring labels for the service

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #3840

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/metrics-server/templates/svc.yaml [hunk 1]
```diff
@@ -3,8 +3,11 @@ kind: Service
 metadata:
   name: {{ template "metrics-server.fullname" . }}
   labels: {{- include "metrics-server.labels" . | nindent 4 }}
+  {{- if .Values.service.labels }}
+  {{- include "metrics-server.tplValue" (dict "value" .Values.service.labels "context" $) | nindent 4 }}
+  {{- end }}
   {{- if .Values.service.annotations }}
-  annotations: {{- include "metrics-server.tplValue" (dict "value" .Values.service.annotation "context" $) | nindent 4 }}
+  annotations: {{- include "metrics-server.tplValue" (dict "value" .Values.service.annotations "context" $) | nindent 4 }}
```
Comment (michelzanini) [inline bitnami/metrics-server/templates/svc.yaml:10]: We are also fixing a bug here. It should be `annotations` not `annotation`... so it's fixed now with this change.

**Review by michelzanini — APPROVED** [review]
We are also fixing a bug here. It should be annotations not annotation... so it's fixed now with this change.

**Review by mohanidragel — APPROVED** [review]

**Review by thecosmicfrog — APPROVED** [review]

File: bitnami/metrics-server/templates/svc.yaml (hunk 1 above)
Comment (andresbono) [reply bitnami/metrics-server/templates/svc.yaml:10]: Good catch!

**Review by andresbono — APPROVED** [review]
LGTM

**Comment by andresbono** [comment]
Thanks for your contribution!!

**Pull request merged**